Makefile

```diff
@@ -132,6 +132,7 @@ _builtinplugins = \
 	ImageTagTransformer.go \
 	LabelTransformer.go \
 	LegacyOrderTransformer.go \
```

Comment:
How difficult would it be to tie the legacy ordering flag to the sortOrderTransformer while preserving existing behavior, and remove all the LegacyOrderTransformer code? Is this something we could do quickly and easily? (If so, it doesn't necessarily have to be done in this PR, but I'd like a sense of how long we need to keep the old code around.)